openfl-tutorials/experimental/workflow/405_MNIST_FederatedEvaluation.ipynb

```diff
@@ -0,0 +1,471 @@
+    "            input.agg_validation_score for input in inputs) / len(inputs)\n",
+    "        print(f'Average aggregated model validation values = {self.aggregated_model_accuracy}')\n",
+    "        self.current_round += 1\n",
+    "        if self.current_round < self.rounds:\n",
+    "            self.next(self.aggregated_model_validation,\n",
+    "                      foreach='collaborators', exclude=['private'])\n",
+    "        else:\n",
+    "            self.next(self.end)\n",
+    "\n",
+    "    @aggregator\n",
+    "    def end(self):\n",
+    "        print(f'This is the end of the flow')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "714ab527",
+   "metadata": {},
+   "source": [
+    "Now lets setup the participants in similar fashion as basic learning/training tutorial but notice the difference in the setup below since we are doing only evaluation there is no need to configure train data and targets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "forward-world",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup participants\n",
+    "aggregator = Aggregator()\n",
+    "aggregator.private_attributes = {}\n",
+    "\n",
+    "# Setup collaborators with private attributes\n",
+    "collaborator_names = ['Portland', 'Seattle', 'Chandler','Bangalore']\n",
+    "collaborators = [Collaborator(name=name) for name in collaborator_names]\n",
+    "for idx, collaborator in enumerate(collaborators):\n",
+    "    local_test = deepcopy(mnist_test)\n",
+    "    local_test.data = mnist_test.data[idx::len(collaborators)]\n",
+    "    local_test.targets = mnist_test.targets[idx::len(collaborators)]\n",
+    "    collaborator.private_attributes = {\n",
+    "            'train_loader': None,\n",
+    "            'test_loader': torch.utils.data.DataLoader(local_test,batch_size=batch_size_test, shuffle=True)\n",
+    "    }\n",
+    "\n",
+    "local_runtime = LocalRuntime(aggregator=aggregator, collaborators=collaborators, backend='single_process')\n",
+    "print(f'Local runtime collaborators = {local_runtime.collaborators}')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "278ad46b",
+   "metadata": {},
+   "source": [
+    "Now that we have our evaluation flow and runtime defined, let's run the experiment! Since its evaluation we need to run it only for one round of validation and for that first we will load a pre-trained model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a175b4d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://github.com/securefederatedai/openfl/raw/refs/heads/develop/openfl-tutorials/experimental/workflow/pretrainedmodels/cnn_mnist.pth cnn_mnist.pth\n",
+    "model = Net()\n",
+    "model.load_state_dict(torch.load('cnn_mnist.pth'))\n",
+    "best_model = model\n",
+    "flflow = FederatedEvaluationFlow(model, rounds=1, checkpoint=True)\n",
```

Comment:
`rounds=1` is already the default for the evaluation flow, so let's not set it here additionally